reservoir-backend.js

```diff
@@ -103,6 +115,9 @@ function ReservoirBackend(options) {
     self.dropCount = {};
     self.logCount = {};
     self.samplingDecision = null;
+
+    var keys = Object.keys(options);
```

Comment:
Remember, this is the reservoir, not the top level Larch -- there's no such thing as a DebugReservoir. I think it's a pretty cool guard for a public interface.